bylaws.md

```diff
@@ -103,10 +103,11 @@ The Fraternity of Lambda Alpha Nu exists to cultivate a strong community around
 	   The Treasurer’s primary responsibility shall be to present a written statement of accounts and transactions to the board on a semesterly basis.
 	   In the case of financial difficulty, the Treasurer shall be responsible for notifying the Board immediately and assisting in the design and execution of recovery efforts.
 5. Terms of Office:
-    1. Elected Officer posistions shall be elected on towards the end of the spring semester on a yearly basis.
+    1. Elected Officer positions shall be elected upon on the 2nd to last scheduled meeting of the academic year.
```

Comment:
Yes, that would be grammatical and understandable.

Another grammatical example would be replacing "upon on" with "during".